chapter-04/recipe-08/using-cost-example/test/a.py

```diff
@@ -0,0 +1 @@
+../../without-cost-example/test/a.py
```

Comment:
I now see why it was failing! Silly me!

Reply:
Still failing:
```
  Test project C:/projects/cmake-cookbook-no-symlinks/chapter-04/recipe-08/using-cost-example/build-2018-06-16-20-34-17
        Start 10: j
   1/10 Test #10: j ................................   Passed    4.61 sec
        Start  9: i
   2/10 Test  #9: i ................................   Passed    3.61 sec
        Start  8: h
   3/10 Test  #8: h ................................   Passed    2.60 sec
        Start  5: e
   4/10 Test  #5: e ................................   Passed    1.60 sec
        Start  6: f
   5/10 Test  #6: f ................................***Failed    0.10 sec
        Start  7: g
   6/10 Test  #7: g ................................***Failed    0.10 sec
        Start  1: a
   7/10 Test  #1: a ................................   Passed    0.59 sec
        Start  2: b
   8/10 Test  #2: b ................................***Failed    0.10 sec
        Start  3: c
   9/10 Test  #3: c ................................***Failed    0.10 sec
        Start  4: d
  10/10 Test  #4: d ................................***Failed    0.10 sec
  
  50% tests passed, 5 tests failed out of 10
  
  Total Test time (real) =  13.54 sec
  
  The following tests FAILED:
  	  2 - b (Failed)
  	  3 - c (Failed)
  	  4 - d (Failed)
  	  6 - f (Failed)
  Errors while running CTest
  	  7 - g (Failed)
```

Reply:
Strange ... back to the drawing board.

Reply:
Aha - it seems that symlinks to symlinks do not get unrolled. I will flatten it more.